lib/media/streaming_engine.js

```diff
@@ -2026,7 +2010,38 @@ shaka.media.StreamingEngine = class {
         const eventName = shaka.util.FakeEvent.EventName.Emsg;
         const data = (new Map()).set('detail', emsg);
         const event = new shaka.util.FakeEvent(eventName, data);
+        // A user can call preventDefault() on a cancelable event.
+        event.cancelable = true;
+
         this.playerInterface_.onEvent(event);
+
+        if (event.defaultPrevented) {
+          // If the caller uses preventDefault() on the 'emsg' event, don't
+          // process any further, and don't generate an ID3 'metadata' event
+          // for the same data.
```

Comment:
Maybe state this in the @exportDoc so users know the behavior of preventDefault() here.

Reply:
Done.  The revision dismissed your approval, though.  Can you take another look real quick?